server/docs/ah-commando-doc.yml

```diff
@@ -139,7 +139,7 @@ paths:
   /users/google:
     get:
       tags:
-        - Social user
+        - Users
```

Comment:
👍🏽However, can you confirm that the same change is not needed for the facebook login?

Reply:
Thanks